client/src/components/Login.tsx

```diff
@@ -139,16 +137,29 @@ const LoginWithoutI18n = (props: LoginProps) => {
   const renderFooter = () => {
     return (
       <div className="building-page-footer">
-        <div className="privacy-modal">
-          <Trans>Your information is secure</Trans>
-          <button
-            className="info-icon"
-            onClick={() => setShowInfoModal(true)}
-            aria-label={i18n._(t`Learn more about how we use your data`)}
-          >
-            <InfoIcon />
-          </button>
-        </div>
+        {isRegisterAccountStep && (
+          <span className="privacy-links">
+            <Trans>
+              Your privacy is important to us. Read our{" "}
+              <a
+                target="_blank"
+                rel="noopener noreferrer"
+                href="https://www.justfix.org/en/privacy-policy/"
+              >
+                Privacy Policy
+              </a>{" "}
+              and{" "}
+              <a
+                target="_blank"
+                rel="noopener noreferrer"
+                href="https://www.justfix.org/en/terms-of-use/"
+              >
+                Terms of Service
+              </a>
+              .
+            </Trans>
+          </span>
```

Comment:
this might be a question for Leslie, but I wonder if an external link icon would be helpful to add to the end of `privacy-links` span. not only to set the expectation that it will open a new tab, but also to differentiate it from the "Login" link below with the same link styling but has a different behavior. 

<img width="158" alt="Screenshot 2024-03-06 at 6 38 45 PM" src="https://github.com/JustFixNYC/who-owns-what/assets/17574626/dc8a1760-758e-43a4-a2bd-2ae3d8dad6f7">

Reply:
Oh yeah leslie and steve were talking about usage of this icon recently and this is a great example of it - I don't remember what they landed on but lets check in about it today

Reply:
We could make log in a standalone internal link, either below or to the right of "Already have an account?"